src/blas/impl/KokkosBlas_util.hpp

```diff
@@ -45,7 +45,121 @@
 #ifndef KOKKOS_BLAS_UTIL_HPP
 #define KOKKOS_BLAS_UTIL_HPP
 
+#include "Kokkos_ArithTraits.hpp"
+
+// TPL macros
+#if defined(KOKKOSKERNELS_ENABLE_TPL_MKL)
```

Comment:
Should also go in TPL directory

Reply:
Applied on https://github.com/kokkos/kokkos-kernels/pull/1433/commits/8d80d3553c91060a7843ef805855ec53beedaf83 with moving the MKL impl.